modules/ai/assets/js/editor/ai-get-started-connect.js

```diff
@@ -19,7 +19,7 @@ function isElementInViewport( el ) {
 	);
 }
 
-export const AiGetStartedConnect = ( { onClose } ) => {
+export const AiGetStartedConnect = ( { onClose, origin } ) => {
```

Comment:
`origin` is a URL property (https://developer.mozilla.org/en-US/docs/Web/API/URL/origin).

Consider rename to `source`.